pyriemann/utils/mean.py

```diff
@@ -641,12 +646,10 @@ def maskedmean_riemann(covmats, masks, tol=10e-9, maxiter=50, init=None,
     else:
         C = init
 
-    k = 0
     nu = 1.0
     tau = np.finfo(np.float64).max
     crit = np.finfo(np.float64).max
-    while (crit > tol) and (k < maxiter) and (nu > tol):
-        k = k + 1
+    for k in range(maxiter):
```

Comment:
```suggestion
    for _ in range(maxiter):
```
as I think k is not used